lambdaclass/mirra_backend
# [GH-669] implement kenzu basic

## Motivation

Kenzu's basic will consist in a combo of skills executed by the same button

Closes #669

## Summary of changes

- Added new cooldown mechanism "combo"

## How to test it?

We don't have kenzu in the game yet, this will be the first iteration of this cooldown mechanism and won't affect the game. to test this replace a character skill for the "kenzu_quickstrike" skill, then play that character in [This](http://localhost:3000/) link and press the button of the skill you changed:
"i" for 1
"o" for 2
"p" for 3

if you press it repeatedly you should see that the skill you perform changes, cycling between:
1. Valtimer's basic
2. H4ck's basic
3. Valtimer's ultimate
### Disclaimer: these skills are placeholders since they are the most notorious ones in the game client for the sake of testing

If you let enough time passes between the first skill you should see that one again





## Checklist
